Gemfile

```diff
@@ -1,4 +1,4 @@
 source 'https://rubygems.org'
 gemspec
 
-gem 'rb-readline'
+gem 'rb-readline', '~> 0.5.5'
```

Comment:
Since we weren't constraining this before we should just make it >= 0.5.5.

Reply:
Well I wanted to be conscious of not breaking anyone if they release something other than a patch. I am ok with restricting it to greater than if we want.